myuw/test/dao/student_profile.py

```diff
@@ -0,0 +1,54 @@
+from django.test import TestCase
```

Comment:
If this is renamed to test_student_profile.py, it will be auto-discovered, and the addition to tests.py won't be needed.